ui/src/services/segmenter/SegmenterScope.js

```diff
@@ -2,11 +2,11 @@ export const getSegmenterScope = (scope) => {
   const status = {
     project: {
       label: "Project",
-      color: "primary",
+      color: "#07C",
```

Comment:
Do primary / success no longer work for this?

Reply:
Ah I made a mistake with the changes here; I thought the primary colours weren't working with `primary`/`success` specified because they seemed to be of a paler shade (so I changed them manually to the brighter ones we see in other parts of the new EUI theme):

![Screenshot 2022-09-13 at 10 40 40 AM](https://user-images.githubusercontent.com/36802364/189796591-12893fe2-20e4-4d61-afad-8686c2d48e31.png)

But after checking out the [docs](https://elastic.github.io/eui/#/display/badge) again for `EuiBadge`, it seems like they are indeed of the intended shade:

![Screenshot 2022-09-13 at 10 47 16 AM](https://user-images.githubusercontent.com/36802364/189796711-c6ddaf1b-d61c-463a-ab40-206e6f0bf5cf.png)

I'll change these values back to `primary`/`success` in this case then 😅

Reply:
Yeah the default color palette is fine. Not sure why the Eui team decided to keep the old colors for some components like Text and this one..